CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+4.1.2 / 2019-06-12
```

Comment:
This was not released yet, so don't set the date - instead use "Work in progress"